node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -72,14 +72,12 @@ object Main {
   /**
     * Starts RNode instance
     * @param options command line options
-    * @param scheduler main scheduler
     * @param console console
     */
   private def startNode(options: commandline.Options)(
       implicit
-      scheduler: Scheduler,
       console: ConsoleIO[Task]
-  ): Unit = {
+  ): Task[Unit] = {
```

Comment:
Is there a particular need to use Task here or benefit? Or it's more to make code consistent?

Reply:
Yes, we want to compose all of our `Task` programs and only execute `Task#unsafeRunSync` once, when we really want to execute the program.